lms/djangoapps/student_profile/views.py

```diff
@@ -87,9 +86,17 @@ def learner_profile_context(request, profile_username, user_is_staff):
             'has_preferences_access': (logged_in_user.username == profile_username or user_is_staff),
             'own_profile': own_profile,
             'country_options': list(countries),
+            'find_courses_url': marketing_link('COURSES'),
             'language_options': settings.ALL_LANGUAGES,
+            'badges_logo': staticfiles_storage.url('certificates/images/backpack-logo.png'),
```

Comment:
Staticfiles can be overwritten by individual platform managers. I don't know how it's commonly done for edX's case, but I know it's a feature of Django's static file handling.